test/show_test.jl

```diff
@@ -222,7 +222,7 @@ let chnlout, sws, sws1, sws2
     io = IOBuffer()
     show(io, sws)
     output = String(take!(io))
-    @test output == "WebSockets.ServerWS(handler=h(r), wshandler=w(s)).out:Channel{Any}(sz_max:2,sz_curr:2) "
+    @test output == "WebSockets.ServerWS(handler=h(r), wshandler=w(s)).out:Channel{Any}(2) "
```

Comment:
Base.Channel no longer maintains a sz_curr field, so it only displays sz_max without naming it.